pkg/service/repair/service.go

```diff
@@ -429,6 +453,30 @@ func (s *Service) Repair(ctx context.Context, clusterID, taskID, runID uuid.UUID
 	return nil
 }
 
+func (s *Service) validateClusterLoadAgainstThreshold(ctx context.Context, client *scyllaclient.Client, hosts []string) error {
+	if s.config.MaxLoadThreshold >= 100 {
+		return nil
+	}
+	notify := func(err error, wait time.Duration) {
+		s.logger.Info(ctx, "Current load of one of the nodes is above the configured threshold...", "error", err, "wait", wait)
+	}
+
+	return retry.WithNotify(ctx, func() error {
+		for _, h := range hosts {
+			load, err := client.CurrentLoad(ctx, h)
+			if err != nil {
+				return retry.Permanent(errors.Wrap(err, "node's load metric"))
+			}
+			if load >= s.config.MaxLoadThreshold {
+				return errors.Errorf("load %.2f on %s node is above configured threshold equal to %.2f",
+					load, h, s.config.MaxLoadThreshold)
```

Comment:
What if the load never goes down?

Reply:
I think we should. If the cluster is underprovisioned, it's better to sacrifice performance than to sacrifice data correctness or disaster recovery. It's easy to add more nodes, not so easy to recover from corruption or data loss.